<?xml version="1.0"?>
<launch>

  <!-- Move base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
  
    <rosparam file="$(find nav)/teb/tfc/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find nav)/teb/tfc/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find nav)/teb/tfc/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav)/teb/tfc/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav)/teb/tfc/base_global_planner_param.yaml" command="load" />
    <rosparam file="$(find nav)/teb/tfc/move_base_params.yaml" command="load" />
    <rosparam file="$(find nav)/teb/tfc/RT.yaml" command="load" />

    <remap from="cmd_vel" to="cmd_vel"/>
    <remap from="odom" to="camera/odom"/>
    <remap from="scan" to="lidar1/scan"/>
    <!--remap from="map" to="rtabmap/map"/-->

    <!--param name="base_global_planner" type="string" value="navfn/NavfnROS" />
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/-->

  </node>


</launch>
